openshift-tools/pkg/discover-etcd-initial-cluster/initial-cluster.go

```diff
@@ -200,13 +200,14 @@ func (o *DiscoverEtcdInitialClusterOptions) Run() error {
 
 // TO DO: instead of archiving, we should remove the directory to avoid any confusion with the backups.
 func archiveDataDir(sourceDir string) error {
-	targetDir := filepath.Join(sourceDir+"-removed-archive", time.Now().Format(time.RFC3339))
+	targetDir := filepath.Join(sourceDir + "-removed-archive" + strings.Replace(time.Now().Format(time.RFC3339), ":", "", -1))
```

Comment:
This will do, but you could also do:

time.Now().Format("yyyyMMddHHmmss")

:-).

Reply:
I tried locally, this was not working 

```
[alpatel@alpatel etcd-v3.3.17-linux-amd64]$ ./temp
attempting to archive /home/alpatel/playground/etcd-v3.3.17-linux-amd64/default.etcd/member to /home/alpatel/playground/etcd-v3.3.17-linux-amd64/default.etcd/member-removed-archiveyyyyMMddHHmmss
```

Reply:
Sorry I meant:
```
[skolicha@localhost timet]$ cat timet.go
package main

import (
    "fmt"
    "time"
)

func main() {
    fmt.Println(time.Now().Format("2006-01-02-030405"))
}
```

Reply:
```
[skolicha@localhost timet]$ ./timet
2020-02-27-121544
[skolicha@localhost timet]$ ./timet
2020-02-27-121546
[skolicha@localhost timet]$ ./timet
2020-02-27-121601
```